components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -304,10 +304,13 @@ void AtmosphereOutput::register_views()
   for (auto const& name : m_fields)
   {
     auto field = m_field_repo->get_field(name, m_grid_name);
+    auto ts    = field.get_header().get_tracking().get_time_stamp();
+    EKAT_REQUIRE_MSG (ts.is_valid(), "Error in output, field " + name + " has not been set in the field manager yet");
```

Comment:
Well, the field _has_ been set in the FM. It's just that nobody has written on it yet...